test-mktemp

```diff
@@ -37,7 +37,7 @@
 
 
 testfun () {
-	TEMP_FILE=`eval $MKTEMP 2>/dev/null`
+	TEMP_FILE=`eval $MKTEMP $1 2>/dev/null`
```

Comment:
Do you know, if `$1` was missing by accident before?
How does this change the behaviour of the test?

Reply:
At that time, the mktemp command `-P' switch was not prepared, so we submitted a workaround, but now the problem has been solved.
The foundation is the source code [commit log](https://cgit.freebsd.org/src/commit/usr.bin/mktemp/mktemp/mktemp.c?id=AC6F924E1EBD983227E906EBA5EEDE).

Therefore, there is no need to check only FreeBSD.